tests/unit/packages/test_deb.py

```diff
@@ -191,6 +191,22 @@ def test_get_package_fetch_error(self, tmpdir, fake_apt_cache):
             )
         assert raised.value.message == "foo"
 
+    def test_unpack_stage_packages_dont_normalize(self, tmpdir, mocker):
+        packages_path = Path(tmpdir, "pkg")
```

Comment:
maybe add a not that an empty directory for stage-packages is being created

Reply:
There's a note about it being empty right before the actual directory creation in lines 200-201.